assets/src/app/pipes/job_status_icon.pipe.ts

```diff
@@ -14,6 +14,7 @@ export class JobStatusIconPipe implements PipeTransform {
       { id: 'completed', name: 'done' },
       { id: 'processing', name: 'refresh'},
       { id: 'error', name: 'clear'},
+      { id: 'queued', name: ''},
```

Comment:
Never used!??

Reply:
yes, on workflow view. It will remove the bad icon.